src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -69,8 +70,35 @@ public ModelStateDictionary ModelState
             }
         }
 
+        /// <summary>
+        /// Gets or Sets the <see cref="Mvc.ActionContext"/> object for execution of action which has been selected
```

Comment:
Be brief with documentation when there's nothing interesting to say. You can just say `Gets or sets the <see cref="Mvc.ActionContext" />.`

The extra stuff you added beyond that isn't really accurate and so it doesn't add value.